tests/test_kem.c

```diff
@@ -1,4 +1,3 @@
-#include <stdbool.h>
```

Comment:
Out of curiosity, why did you delete this?

Reply:
Wasn't needed in test_sig.c, so after comparing with test_kem.c, I figured it wasn't needed there either.